tools/lxdclient/client.go

```diff
@@ -271,7 +271,7 @@ func verifyDefaultProfileBridgeConfig(client *lxd.Client, networkAPISupported bo
 		return "", errors.Trace(err)
 	}
 
-	// If the default profile doesn't have eth0 in it, then the user has messed
+	// If the default profile has eth0 in it, then the user has messed
```

Comment:
Yeah, I'll just delete it all together. It was wrong before, and it's wrong now :)